From 8ff4b056e8fd371dca868f8967b36c6e9a4590ab Mon Sep 17 00:00:00 2001
From: Emmanuel Gil Peyrot <linkmauve@linkmauve.fr>
Date: Fri, 10 Jul 2020 05:06:38 +0200
Subject: [PATCH 28/65] Replace the last GtkStock in overwrite dialog

Also add a mnemonic for better keyboard access to the overwrite button.
---
 src/gui/engine.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gui/engine.cc b/src/gui/engine.cc
index c66279e..ff06074 100644
--- a/src/gui/engine.cc
+++ b/src/gui/engine.cc
@@ -556,8 +556,8 @@ void save_as(GtkWidget *mainwindow)
                                   "A file named <i><tt>%s</tt></i> already exists in folder <tt>%s</tt>.\n\nDo you want to overwrite it?",
                                   basename(buffer1), dirname(buffer2));
         gtk_dialog_add_buttons(GTK_DIALOG(dialog), 
-				  GTK_STOCK_CANCEL, GTK_RESPONSE_CANCEL,
-				  "Overwrite", GTK_RESPONSE_ACCEPT,
+				  "_Cancel", GTK_RESPONSE_CANCEL,
+				  "_Overwrite", GTK_RESPONSE_ACCEPT,
                                   NULL);
         proceed = (gtk_dialog_run (GTK_DIALOG (dialog)) == GTK_RESPONSE_ACCEPT);
         gtk_widget_destroy (dialog);
-- 
2.33.1

